fetch_grasp_suggestion/RankedGrasp[] grasps  # ranked list of grasps
int32 object_index                           # index of object to which grasps apply